swagger-java-wes-client/pom.xml

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~    Copyright 2019 OICR
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>swagger-java-wes-client</artifactId>
+    <packaging>jar</packaging>
+    <name>swagger-java-wes-client</name>
+
+    <parent>
+        <artifactId>dockstore</artifactId>
+        <groupId>io.dockstore</groupId>
+        <version>1.6.0-beta.2-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.swagger.codegen.v3 </groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>3.0.5</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <!--
+                                        <inputSpec>https://raw.githubusercontent.com/ga4gh/workflow-execution-service-schemas/1.0.0/openapi/workflow_execution_service.swagger.yaml</inputSpec>
+                             -->
+                            <inputSpec>${project.basedir}/src/main/resources/workflow-execution_service-1.0.0-oas3-swagger.yaml</inputSpec>
```

Comment:
In a follow-up, please rename `swagger-java-wes-client` to `openapi-java-wes-client` since this actually using openapi 3 already

Reply:
https://ucsc-cgl.atlassian.net/browse/DOCK-574